bin/main.rb

```diff
@@ -1 +1,59 @@
-puts "Hello World!"
\ No newline at end of file
+# Set up game
+puts "Welcome!"
+puts "Player 1, What is your name?"
+@player1 = gets.chomp
+puts "Now, Player 2, What is your name?"
+@player2 = gets.chomp
+puts "Let's play! #{@player1} #{@player2}"
+continue_game = true
+
+# While nobody has won
+while continue_game
+  # get input from player 1
+  puts "Choose a number between 1-9 #{@player1}"
+  @player1_input = gets.chomp.to_i
+  # Make sure user has the correct input
+  if @player1_input.is_a?(Integer) && @player1_input > 0 && @player1_input <= 9
```

Comment:
Don't you think `@player1_input.is_a?(Integer)` is redundant, since you have already converted the player input to integer in L14 `gets.chomp.to_i`? :thinking: 

`@player1_input.is_a?(Integer)` will always return true.

Reply:
Thank you for pointing that out @bolah! That was part of a previous method we were using and overlooked it. Really looking forward to learning RSpec to catch those before code reviews! 😆